recipes-devtools/rust/rust-snapshot-1.20.0.inc

```diff
@@ -24,3 +24,4 @@ SRC_URI[rust-std-snapshot.md5sum] = "2bff47764df01c99f349908601c10478"
 SRC_URI[rust-std-snapshot.sha256sum] = "5905803e8a127f656bf253978692f0d6cf6c9206c527e4d6d7e981980618d1b6"
 SRC_URI[cargo-snapshot.md5sum] = "63aa861b029eec9f559f4fb5a10c287d"
```

Comment:
Changing `name=` above without changing the `SRC_URI[name]` here seems like it will break 1.20.

Reply:
You are right. That file was accidentally included. I've removed it now.